scm/github/testdata/hooks/delete.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
I don't think we need this file, right? We probably need a push payload that is of the delete variety though... with a test 😄